test/realizations/catchments/Bmi_C_Adapter_Test.cpp

```diff
@@ -226,9 +238,12 @@ TEST_F(Bmi_C_Adapter_Test, Update_0_g) {
     adapter.Finalize();
 }
 
-/** Test that the update function works fails after exceeding end time. */
+/** Test that the update function fails after exceeding end time. */
+/* This actually doesn't work currently, but this is due to a change in the way the example model is implemented.
+ * Re-add this test if/once we have a model that doesn't work beyond its time step.
```

Comment:
Does this test actually need to be diabled?  This comment makes it seem so, but the test still is active.  We can diable the test by prepending `DISABLED_` to the test name, i.e.
`TEST_F(Bmi_C_Adapter_Test, DISABLED_Update_0_h) {`

Reply:
I'd forgotten about the `DISABLED_` capability.  I'm going to change this one and a few others that I'd commented out for similar reasons.